### PR TITLE
add info that backup file can also be uploaded to aura

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1759.surge.sh

Updated pages:
- [Upload to Neo4j Aura](https://neo4j-docs-operations-1759.surge.sh/operations-manual/5/tools/neo4j-admin/upload-to-aura)

File: modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc [hunk 1]
```diff
@@ -84,7 +84,7 @@ The `neo4j-admin database upload` command has the following options:
 |
 
 |--from-path=<path>
-|_/path/to/directory-containing-dump_ Path to a directory containing a database dump to upload.
+|_/path/to/directory-containing-dump-or-backup_ Path to a directory containing a database dump or backup file to upload.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc:87]: These parts are tested, so we cannot change the description here without changing it in the code first. Now our test fails with the message: 
```
AssertionError: Mismatch in description for option --from-path=<path>.
07:40:12   E   == Docs ==
07:40:12   E   /path/to/directory-containing-dump-or-backup Path to a directory containing a database dump or backup file to upload.
07:40:12   E   == Command ==
07:40:12   E   /path/to/directory-containing-dump Path to a directory containing a database dump to upload.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1759.surge.sh

Updated pages:
- [Upload to Neo4j Aura](https://neo4j-docs-operations-1759.surge.sh/operations-manual/5/tools/neo4j-admin/upload-to-aura)

**Review by jedops — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1759.surge.sh

Updated pages:
- [Upload to Neo4j Aura](https://neo4j-docs-operations-1759.surge.sh/operations-manual/5/tools/neo4j-admin/upload-to-aura)

**Pull request merged**